docs-source/content/userguide/managing-domains/domain-events.md

```diff
@@ -49,10 +54,10 @@ Each operator-generated event in a domain namespace contains the following field
  *  `reason`:  Short, machine understandable string that gives the reason for the transition to the object's current status.
  *  `message`:  String that describes the details of the event.
  *  `involvedObject`:  `V1ObjectReference` object that describes the Kubernetes resources with which this event is associated.
-    *  `name`:  String that describes the name of the resource, either a domain resource, in which case the name is the `domainUID`, or a namespace, in which case the name is the name of the namespace.
-    *  `namespace`:  String that describes the namespace of the event, which is the namespace of the domain resource for a domain event.
-    *  `kind`:  String that represents the REST resource this object represents. The value is `Domain` for a domain event, and `Namespace` for a namespace event.
-    *  `apiVersion`:  String that describes the `apiVersion` of the involved object, which is the `apiVersion` of the domain resource, for example, `weblogic.oracle/v8`, for a domain event, and is not set for a namespace event.
+    *  `name`:  String that describes the name of the resource that the event is associated with. It may be the `domainUID`, the name of the namespace that the operator watches, or the name of the operator pod.
```

Comment:
resource that the event is associated with. -> resource with which the event is associated.